packages/wdio-local-runner/src/worker.ts

```diff
@@ -118,6 +119,8 @@ export default class WorkerInstance extends EventEmitter implements Workers.Work
             stdio: ['inherit', 'pipe', 'pipe', 'ipc']
         })
 
+        await sleep(1 * 1000)
```

Comment:
I don't think it is a good idea to have an implicit sleep here, most of the time it probably won't take a second and the problem still occurs if it takes longer for whatever reason.